src/Microsoft.DotNet.Cli.Utils/CommandResolution/PackagedCommandSpecFactory.cs

```diff
@@ -104,7 +107,18 @@ internal PackagedCommandSpecFactory(Action<string, IList<string>> addAdditionalA
                     nugetPackagesRoot,
                     runtimeConfigPath);
             }
-            
+
+            // Reporter.Verbose.WriteLine(string.Format(
+            //     LocalizableStrings.AttemptingToFindCommand,
+            //     PackagedCommandSpecFactoryName,
+            //     commandName,
+            //     toolLibrary.Name));
+Console.WriteLine($"appconfig: {appConfigPath} -> {commandPath}.config");
```

Comment:
Can we put the Reporter back please? We don't want this showing every time someone tries to run a tool.